src/main/java/it/gov/pagopa/ranking/repository/OnboardingRankingRequestsRepository.java

```diff
@@ -9,4 +9,6 @@
 public interface OnboardingRankingRequestsRepository extends MongoRepository<OnboardingRankingRequests,String> {
 
     List<OnboardingRankingRequests> findByInitiativeId(String initiativeId, Pageable pageable);
+
+    long countByInitiativeId(String initiativeId);
```

Comment:
this is not necessary, you can write here directly
Page<OnboardingRankingRequests> findByInitiativeId(String initiativeId, Pageable pageable);

Spring will provide to perform the query and the count together